azext_iot/tests/test_iot_ext_int.py

```diff
@@ -71,6 +71,33 @@ def test_hub(self):
             expect_failure=True,
         )
 
+        # Test 'az iot hub connection-string show'
+        conn_str_pattern = r'^HostName={0}.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey='.format(
+            LIVE_HUB)
+        conn_str_eventhub_pattern = r'^Endpoint=sb://'
+        self.cmd('iot hub connection-string show -n {0}'.format(LIVE_HUB), checks=[
```

Comment:
Can we also please add tests for showing connection-string by resource group or by sub? We know the sub or group should have at least 1  IoT Hub and the count of iothubs in a group should be <= count of iothubs in sub.

Reply:
Added new test case